hwdb.d/60-keyboard.hwdb

```diff
@@ -1,10 +1,11 @@
 # This file is part of systemd.
 #
-# This file contains 3 types of metadata to apply to keyboards and
+# This file contains 4 types of metadata to apply to keyboards and
 # keyboard-like input devices:
 # - Key mapping
 # - Hard-coded layouts
 # - Absence of modifier LEDs
+# - Chassis keys (usually to ignore for unideling the device)
```

Comment:
```suggestion
# - Chassis keys (usually to ignore for unidling the device)
```

Reply:
The same typo also in the commit message.